json-rpc.md

```diff
@@ -6,21 +6,21 @@ In Flow Launcher, we use JSON-RPC as a **local** procedure call protocol to bind
 
 So we need to build a **common API** between Flow and Plugin.
 
-![JSON RPC](/assets/jsonrpc.png)
+![JSON-RPC](/assets/jsonrpc.png)
 
 ### Example
 
 - `-->` denotes data sent to FLow.
 - `<--` denotes data coming from Flow.
 
-```json
+```js
```

Comment:
JSON highlighter is not included, so this block was not highlighted, but I don't think the JSON highlighter is needed in this case. JSON is basically JS, so the JS highlighter can be reused for them.